pkg/kv/kvserver/store_raft.go

```diff
@@ -32,6 +32,11 @@ import (
 	"go.etcd.io/etcd/raft/v3/raftpb"
 )
 
+var (
+	logRaftRecvQueueFullEvery = log.Every(1 * time.Second)
+	logRaftSendQueueFullEvery = log.Every(1 * time.Second)
```

Comment:
If this is happening I'm happy to have it logged every second so that it can't be missed, since these are probably the most important thing going on in the cluster (unless disks are straight-up stuck).